std/datetime.d

```diff
@@ -2464,23 +2502,77 @@ public:
 
 
     /++
+        Converts from unix time (i.e. seconds from midnight, January 1st, 1970
+        in UTC) to a $(LREF SysTime).
+
+        On most systems, unix time is equivalent to time_t, but that depends on
+        the C implementation as the exact representation of time_t is
+        implementation-defined (e.g. it does not appear that the Digital Mars C
+        runtime uses unix time for time_t; it appears to apply the local DST to
+        time_t).
```

Comment:
I think this explanation can be shorter (and less committed to a specific behavior):

```
Note: unix time is not necessarily equivalent to time_t resolution or origin on non-unix systems. Keep this in mind when mixing with C calls.
```